include/RAJA/policy/PolicyBase.hpp

```diff
@@ -59,7 +59,7 @@
 namespace RAJA
 {
 
-enum class Policy { undefined, sequential, simd, openmp, target_openmp, cuda };
+enum class Policy { undefined, sequential, simd, openmp, target_openmp, cuda, tbb };
```

Comment:
If we add a Policy, we should also add corresponding type traits (is_tbb_policy)